features/step_definitions/common_steps.rb

```diff
@@ -1,3 +1,42 @@
 When(/^I click on continue$/) do
-  your_case_continued_page.button.click
+  common_page.button.click
+end
+
+Then(/^I should see '([^\"]*)' header$/) do |header|
+  expect(common_page.h1.text).to eq header
```

Comment:
h1 seems like a tag name ?  Should we not use more meaningful names here ?  What would happen if there were more than 1 h1 on the page ?

Reply:
I've added this ticket for future improvements, to add data attributes with more meaningful names. https://tools.hmcts.net/jira/projects/RST/issues/RST-523?filter=allopenissues

For now I will continue to call the h1, h1 for example as it's easier to debug and it's unlikely a non technical person is going to look at page objects or step files.

Suggest a team conversation about this.